tests/agent/test_docker_agent.py

```diff
@@ -898,7 +898,7 @@ def test_docker_agent_is_named_volume_win32(monkeypatch, api, path, result):
             ["some-name:/ctr/path"],
             ["some-name"],
             ["/ctr/path"],
-            {},
+            {"some-name": {"bind": "/ctr/path", "mode": "rw"}},
```

Comment:
is this change in coverage intentional vs adding another full case?

Reply:
The old code was actually just wrong and did not work, so this change is correcting coverage of bad behavior. A bind must be generated alongside a named volume.